share/gpodder/ui/gtk/gpodder.ui

```diff
@@ -442,22 +513,26 @@
                                 <signal name="toggled" handler="on_cbLimitDownloads_toggled" swapped="no"/>
                               </object>
                               <packing>
-                                <property name="left-attach">0</property>
-                                <property name="top-attach">0</property>
+                                <property name="expand">False</property>
+                                <property name="fill">True</property>
+                                <property name="position">0</property>
                               </packing>
                             </child>
                             <child>
                               <object class="GtkSpinButton" id="spinLimitDownloads">
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
                                 <property name="invisible-char">●</property>
+                                <property name="text" translatable="yes">0,5</property>
                                 <property name="adjustment">adjustment1</property>
                                 <property name="climb-rate">1</property>
                                 <property name="digits">1</property>
+                                <property name="value">0.5</property>
```

Comment:
What is this value and the translatable property above it?

Reply:
These were probably added by glade. The latest commit fixes the properties of the SpinButtons.